packages/web-shared/providers/AnalyticsProvider.js

```diff
@@ -55,16 +56,22 @@ export const GET_ANALYTICS_USER = gql`
 export const AnalyticsProvider = ({ children, church }) => {
   const { currentChurch } = useCurrentChurch();
   const { currentUser } = useCurrentUser();
+  const isBot = useIsBot();
```

Comment:
Does this need to be a hook, the call looks synchronous :)

Reply:
I agree, don't really need a hook

Reply:
this is fixed